featuretests/cmd_juju_crossmodel_test.go

```diff
@@ -369,6 +369,11 @@ func (s *crossmodelSuite) assertAddRelationSameControllerSuccess(c *gc.C, otherM
 }
 
 func (s *crossmodelSuite) TestAddRelationSameControllerSameOwner(c *gc.C) {
+	s.addOtherModelApplicationSameOwner(c)
```

Comment:
Not sure why we need these changes

Reply:
Sorry, will clean up (spent some time trying to use this code to add more tests)